src/sentry/conf/server.py

```diff
@@ -21,6 +21,7 @@
 from sentry.conf.types.topic_definition import TopicDefinition
 from sentry.utils import json  # NOQA (used in getsentry config)
 from sentry.utils.celery import crontab_with_minute_jitter
+from sentry.utils.sdk_crashes.config import SDKCrashDetectionConfig
```

Comment:
please move this type into `sentry.conf.types` -- I'm trying to eliminate any external import here since any import cycle breaks all settings typing